lib/Twig/Extensions/Extension/Intl.php

```diff
@@ -55,12 +55,17 @@ function twig_localized_date_filter(Twig_Environment $env, $date, $dateFormat =
         'full' => IntlDateFormatter::FULL,
     );
 
+    $calendarValues = array(
+        'gregorian' => IntlDateFormatter::GREGORIAN,
+        'traditional' => IntlDateFormatter::TRADITIONAL,
```

Comment:
then you should change the PR description to avoid listing others, as they won't be supported using this argument